Alloy/commands/compile/parsers/Ti.UI.Windows.AppBarButton.js

```diff
@@ -0,0 +1,38 @@
+var systemIconList = [
+	'ACCEPT', 'ACCOUNT', 'ADD', 'ADDFRIEND', 'ADMIN', 'ALIGNCENTER', 'ALIGNLEFT',
+	'ALIGNRIGHT', 'ALLAPPS', 'ATTACH', 'ATTACHCAMERA', 'AUDIO',
+	'BACK', 'BACKTOWINDOW', 'BLOCKCONTACT', 'BOLD', 'BOOKMARKS', 'BROWSEPHOTOS', 'BULLETS',
+	'CALCULATOR', 'CALENDAR', 'CALENDARDAY', 'CALENDARREPLY', 'CALENDARWEEK', 'CAMERA', 'CANCEL',
+	'CAPTION', 'CELLPHONE', 'CHARACTER', 'CLEAR', 'CLEARSELECTION', 'CLOCK', 'CLOSEDCAPTION',
+	'CLOSEPANE', 'COMMENT', 'CONTACT', 'CONTACT2', 'CONTACTINFO', 'CONTACTPRESENCE', 'COPY', 'CROP', 'CUT',
+	'DELETE', 'DIRECTIONS', 'DISABLEUPDATES', 'DISCONNECTDRIVE', 'DISLIKE', 'DOCKBOTTOM', 'DOCKLEFT',
+	'DOCKRIGHT', 'DOCUMENT', 'DOWNLOAD', 'EDIT', 'EMOJI', 'EMOJI2',
+	'FAVORITE', 'FILTER', 'FIND', 'FLAG', 'FOLDER', 'FONT', 'FONTCOLOR', 'FONTDECREASE',
+	'FONTINCREASE', 'FONTSIZE', 'FORWARD', 'FOURBARS', 'FULLSCREEN', 'GLOBE', 'GO',
+	'GOTOSTART', 'GOTOTODAY', 'HANGUP', 'HELP', 'HIDEBCC', 'HIGHLIGHT', 'HOME', 'IMPORT', 'IMPORTALL',
+	'IMPORTANT', 'ITALIC', 'KEYBOARD', 'LEAVECHAT', 'LIBRARY', 'LIKE', 'LIKEDISLIKE', 'LINK', 'LIST', 'MAIL', 'MAILFILLED',
+	'MAILFORWARD', 'MAILREPLY', 'MAILREPLYALL', 'MANAGE', 'MAP', 'MAPDRIVE', 'MAPPIN', 'MEMO', 'MESSAGE', 'MICROPHONE',
+	'MORE', 'MOVETOFOLDER', 'MUSICINFO', 'MUTE', 'NEWFOLDER', 'NEWWINDOW', 'NEXT', 'ONEBAR', 'OPENFILE', 'OPENLOCAL',
+	'OPENPANE', 'OPENWITH', 'ORIENTATION', 'OTHERUSER', 'OUTLINESTAR', 'PAGE', 'PAGE2', 'PASTE', 'PAUSE', 'PEOPLE', 'PERMISSIONS',
+	'PHONE', 'PHONEBOOK', 'PICTURES', 'PIN', 'PLACEHOLDER', 'PLAY', 'POSTUPDATE', 'PREVIEW', 'PREVIEWLINK', 'PREVIOUS', 'PRIORITY',
+	'PROTECTEDDOCUMENT', 'READ', 'REDO', 'REFRESH', 'REMOTE', 'REMOVE', 'RENAME', 'REPAIR', 'REPEATALL', 'REPEATONE', 'REPORTHACKED',
+	'RESHARE', 'ROTATE', 'ROTATECAMERA', 'SAVE', 'SAVELOCAL', 'SCAN', 'SELECTALL', 'SEND', 'SETLOCKSCREEN', 'SETTILE', 'SETTING',
+	'SHOP', 'SHOWBCC', 'SHOWRESULTS', 'SHUFFLE', 'SLIDESHOW', 'SOLIDSTAR', 'SORT', 'STOP', 'STOPSLIDESHOW', 'STREET', 'SWITCH',
+	'SWITCHAPPS', 'SYNC', 'SYNCFOLDER', 'TAG', 'TARGET', 'THREEBARS', 'TOUCHPOINTER', 'TRIM', 'TWOBARS', 'TWOPAGE', 'UNDERLINE',
+	'UNDO', 'UNFAVORITE', 'UNPIN', 'UNSYNCFOLDER', 'UP', 'UPLOAD', 'VIDEO', 'VIDEOCHAT', 'VIEW', 'VIEWALL', 'VOLUME', 'WEBCAM',
+	'WORLD', 'ZEROBARS', 'ZOOM', 'ZOOMIN', 'ZOOMOUT'
+];
+
+exports.parse = function(node, state) {
+	return require('./base').parse(node, state, parse);
+};
+
+function parse(node, state, args) {
+	var systemIcon = node.getAttribute('icon');
+	if (systemIconList.indexOf(systemIcon) !== -1) {
```

Comment:
I'd just do:

```
if (systemIcon.indexOf('.SystemIcon.') === -1) {
```

Otherwise if we introduce a new constant you'll get weird behaviour.